src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -569,6 +601,7 @@ def __call__(
         clip_skip: Optional[int] = None,
         callback_on_step_end: Optional[Callable[[int, int, Dict], None]] = None,
         callback_on_step_end_tensor_inputs: List[str] = ["latents"],
+        vae_batch_size: int = 16,
```

Comment:
Let's use naming/logic similar to SVD for batch decoding. 
https://github.com/huggingface/diffusers/blob/73acebb8cfbd1d2954cabe1af4185f9994e61917/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L607

Reply:
This is also used in the vae encode for [animatediff_video2video](https://github.com/huggingface/diffusers/pull/8948/files#diff-ead50799e2d01ae9dae0ebfb14065cc275ebd184171ad5277e39b26d105f1b6cR503) btw, but can rename it that